app/models/feedback.rb

```diff
@@ -32,6 +32,9 @@ class Feedback < ActiveRecord::Base
 
   scope :unsubmitted, -> { where(submitted: false) }
 
+  SELF_REPORTED = 'Self-reported'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.